bindings/pyroot_experimental/cppyy/cppyy-backend/cling/patches/dep_template.diff

```diff
@@ -0,0 +1,59 @@
+diff --git a/src/core/dictgen/src/rootcling_impl.cxx b/src/core/dictgen/src/rootcling_impl.cxx
```

Comment:
Can we resolve those asap? I really don't want to see cling patches in ROOT's sources :-) Please discuss with Wim on whether cling's HEAD of master still needs them.

Reply:
Sure, would you like me to give priority to incorporating Wim's mini-ROOT patches to master, over e.g. adding pythonizations or fixing tests? I will let Wim know so that he can tell us which ones we need.
In the meantime, we can just remove that directory from this PR.